maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java

```diff
@@ -748,10 +748,10 @@ private void validateRawRepositories( ModelProblemCollector problems, List<Repos
 
         for ( Repository repository : repositories )
         {
-            validateStringNotEmpty( prefix, ".id", problems, Severity.ERROR, Version.V20, repository.getId(),
+            validateStringNotEmpty( prefix, prefix2, ".id", problems, Severity.ERROR, Version.V20, repository.getId(),
```

Comment:
This looks inconsistent with the diff below. This one has a trailing dot, the one below does not. Is that correct?